tests/apollo/test_skvbc_dbsnapshot.py

```diff
@@ -158,27 +161,35 @@ def start_replica_cmd_with_operator_and_public_keys(builddir, replica_id):
             "-b", "2",
             "-q", batch_size,
             "-h", "3",
-            "-j", "600",
+            "-j", str(DB_CHECKPOINT_HIGH_WIN_SIZE),
             "-o", builddir + "/operator_pub.pem",
             "--add-all-keys-as-public"]
 
 class SkvbcDbSnapshotTest(ApolloTest):
     __test__ = False  # so that PyTest ignores this test scenario
 
+    @staticmethod
+    async def send_kvs_sequentially(skvbc: 'SimpleKVBCProtocol', kv_count: int, client: 'BftClient' = None):
```

Comment:
This is a copy of a method used in another test class, I meant you should move it to the parent, not copy it.

Reply:
Moved to skvbc.py